### PR TITLE
Add properties for emitting to Datadog to ATC

### PR DESCRIPTION
**Pull request by SHyx0rmZ**
These seemed to be missing.

### CODE REVIEW

File: jobs/atc/spec [hunk 1]
```diff
@@ -467,6 +467,22 @@ properties:
       Environment name to specify for errors emitted to Yeller.
     default: ""
 
+  datadog.agent_host:
+    description: |
+      If configured, detailed metrics will be emitted to the specified Datadog Agent's
+      dogstatsd server.
+    default: ""
```
Comment (vito) [inline jobs/atc/spec]: Don't need a default here since we're using `if_p` in the template.

File: jobs/atc/spec [hunk 2]
```diff
@@ -467,6 +467,22 @@ properties:
...
+
+  datadog.agent_port:
+    description: |
+      Port of the Datadog Agent's dogstatsd server to emit events to.
+    default: 8125
+
+  datadog.prefix:
+    description: |
+      An optional prefix for emitted Datadog events.
+    default: ""
```
Comment (vito) [inline jobs/atc/spec]: Same as above

**Comment by SHyx0rmZ** [comment]
Oh, totally missed the notification for this. Fixed.

**Comment by vito** [comment]
thanks!

**Comment by SHyx0rmZ** [comment]
Thanks for merging!

**Pull request merged**